play.pokemonshowdown.com/js/client-chat-tournament.js

```diff
@@ -673,7 +673,7 @@
 				var numLeaves = 0;
 				var hasLeafAtDepth = [];
 				this.forEachTreeNode(newTree, function (node, depth) {
-					hasLeafAtDepth[depth] || (hasLeafAtDepth[depth] = false);
```

Comment:
Haha I guess the compiler did this and the linter didn't like it.

Reply:
i dont blame it, i dont like it either! i wish i could use `||=` in es3